From 471ee4bb9b461c0555cf46c2ea5c423827ec1369 Mon Sep 17 00:00:00 2001
From: Jan Fajerski <jfajerski@suse.com>
Date: Fri, 8 Feb 2019 09:17:06 +0100
Subject: [PATCH] Do not force the pure Go name resolver

Revert to Go's default mechanism that will decide between the `netgo`
pure-Go implementation and the `netcgo` cgo-based implementation depending
on OS and environment variables.

This allows, among other things, to use Prometheus to scrape mDNS targets.

Signed-off-by: Jan Fajerski <jfajerski@suse.com>
Signed-off-by: Johannes Kastl <git@johannes-kastl.de>
---
 .promu.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.promu.yml b/.promu.yml
index d16bceeed..367a75765 100644
--- a/.promu.yml
+++ b/.promu.yml
@@ -12,10 +12,10 @@ build:
           path: ./cmd/promtool
     tags:
         all:
-            - netgo
             - builtinassets
         windows:
             - builtinassets
+    flags: -mod=vendor -a
     ldflags: |
         -X github.com/prometheus/common/version.Version={{.Version}}
         -X github.com/prometheus/common/version.Revision={{.Revision}}
-- 
2.49.0

